src/pages/docs.tsx

```diff
@@ -21,35 +21,25 @@ import ShellBox from '../components/ShellBox';
 import '../styles/docs.scss';
 import '../styles/article-reader.scss';
 
+const API_DOCS_OBJ_KEYS = ['events', 'methods', 'properties', 'classes'];
+
 function renderArticleOverview(
   obj: ApiDocsObj,
   overview: JSX.Element[] = []
 ): JSX.Element[] {
   const children: JSX.Element[] = [];
-  if (obj.events) {
-    obj.events.map((evt): JSX.Element[] =>
-      renderArticleOverview(evt, children)
-    );
-  }
-  if (obj.methods) {
-    obj.methods.map((method): JSX.Element[] =>
-      renderArticleOverview(method, children)
-    );
-  }
-  if (obj.properties) {
-    obj.properties
-      .filter((o): boolean => o.type !== 'Object')
-      .map((prop): JSX.Element[] => renderArticleOverview(prop, children));
 
-    obj.properties
-      .filter((o): boolean => o.type === 'Object')
-      .map((prop): JSX.Element[] => renderArticleOverview(prop, children));
-  }
-  if (obj.classes) {
-    obj.classes.map((klass): JSX.Element[] =>
-      renderArticleOverview(klass, children)
-    );
-  }
+  API_DOCS_OBJ_KEYS.map(function iterateOverApiDocsObjects(key: string) {
```

Comment:
I would break out these inner functions like this.

```  const removeObjectTypeForProperties = (property: object): boolean => {
    return key === 'properties' && property.type !== 'Object';
  };

  const iterateOverApiDocsObjects = (key: string): void => {
    if (obj[key]) {
      obj[key]
        .filter(removeObjectTypeForProperties)
        .map((docObject: ApiDocsObj) =>
          renderArticleOverview(docObject, children)
        );
    }
  };

  API_DOCS_OBJ_KEYS.map(iterateOverApiDocsObjects);
```

Either way it's a lot more readable then the old code so 👍